doc/move_code/objects_tutorial/sources/Ch1_2/ColorObject.move

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module Tutorial::ColorObject {
+    use Sui::ID::{Self, VersionedID};
+    use Sui::Transfer;
+    use Sui::TxContext::{Self, TxContext};
+
+    struct ColorObject has key {
+        id: VersionedID,
+        red: u8,
+        green: u8,
+        blue: u8,
+    }
+
+    // == Functions covered in Chapter 1 ==
+
+    fun new(red: u8, green: u8, blue: u8, ctx: &mut TxContext): ColorObject {
+        ColorObject {
+            id: TxContext::new_id(ctx),
+            red,
+            green,
+            blue,
+        }
+    }
+
+    public fun create(red: u8, green: u8, blue: u8, ctx: &mut TxContext) {
+        let color_object = new(red, green, blue, ctx);
+        Transfer::transfer(color_object, TxContext::sender(ctx))
+    }
+
+    public fun get_color(self: &ColorObject): (u8, u8, u8) {
+        (self.red, self.green, self.blue)
+    }
+
+    // == Functions covered in Chatper 2 ==
+
+    public fun delete(object: ColorObject, _ctx: &mut TxContext) {
+        let ColorObject { id, red: _, green: _, blue: _ } = object;
+        ID::delete(id);
+    }
+
+    public fun transfer(object: ColorObject, recipient: address, _ctx: &mut TxContext) {
```

Comment:
It's different though. The one from `Transfer::transfer` cannot be used as entry function, while this one can.
Here I was hoping to demo calling this in the wallet.

Reply:
I wonder if this is worth explaining in the text in case others have the same question as Sam. Something like:

We use this rather than `Transfer::transfer` as the latter cannot be used as an [entry function](../../../../../src/build/move#entry-functions).

Reply:
It is explained in the tutorial, though.

Reply:
> It is explained in the tutorial, though.

Beautiful. I see that now I've edited it:
https://github.com/MystenLabs/sui/pull/1361

I also included that entry points link. Thanks!